x-pack/plugins/reporting/server/lib/store/store.ts

```diff
@@ -124,62 +119,28 @@ export class ReportingStore {
     return this.client;
   }
 
-  private async getIlmPolicyManager() {
-    const client = await this.getClient();
-    return IlmPolicyManager.create({ client });
-  }
-
-  private async createIndex(indexName: string) {
```

Comment:
No need to create the index or manually apply index settings in Reporting! That is handled in ES now.